extensions/install.php

```diff
@@ -1241,7 +1245,7 @@ protected function uninstallTranslations($parent)
 						{
 							foreach ($translationTables as $translationTableParams)
 							{
-								if ($extensionOption == $translationTableParams->option)
+								if ($extensionOption === $translationTableParams->option)
```

Comment:
same here with string uppercase and lowercase, it should not be enforced throughout whis whole file, I dont really see the need for it